### DISCUSSION TITLE
Be able to add more than 1 executor to the report

### DISCUSSION THREAD

**Discussion by harmin-parra** [allure-framework/allure2#2578] [Ideas & Feedback]
Hello

Currently, Allure allow the inclusion of just one executor (https://allurereport.org/docs/how-it-works-executor-file/)

![image](https://github.com/allure-framework/allure2/assets/16239820/c37c5a8a-8ebb-4ac4-a669-93a12815a753)

Could it be possible to add more executors to the report?

In my use case, I generate my Allure report from different sources, each source is executed in a different job in the pipeline.

I execute 2 type of tests:
* Web interface tests
* REST api test

Each type of test is executed in a different job in the pipeline because each type of test runs in a different docker image.
I would like to include the information of my two executors in the Allure report.

**Comment by baev** [comment]
You can achieve that by providing a separate `executor.json` file for each `allure-results` and then generating the report specifying multiple results folders:

```bash
$ allure serve path/to/web/allure-results path/to/rest/allure-results
```

<img width="782" alt="Screenshot 2024-05-29 at 18 32 18" src="https://github.com/allure-framework/allure2/assets/2149631/4c47d299-c067-42e5-ac15-d0ace5a884a5">

**Reply by baev** [reply]
<img width="443" alt="Screenshot 2024-05-29 at 18 36 19" src="https://github.com/allure-framework/allure2/assets/2149631/cdc36675-88d6-48d7-9725-d2d50de88f2b">

**Comment by harmin-parra** [comment]
Thanks
